python/lzreposync/src/lzreposync/importUtils.py

```diff
@@ -0,0 +1,98 @@
+import sys
+
+from spacewalk.satellite_tools.syncLib import log2, log
+from spacewalk.server import rhnSQL
+from spacewalk.server.importlib import importLib, mpmSource, packageImport
+from spacewalk.server.importlib.backendOracle import SQLBackend
+from uyuni.common.context_managers import cfg_component
```

Comment:
We moved this import. The old path is still working, but new code should use the new location:
```suggestion
from spacewalk.common.rhnConfig import cfg_component
```

Reply:
Couldn't find `cfg_component` in [spacewalk.common.rhnConfig](https://github.com/uyuni-project/uyuni/blob/prepare-lzreposync-layout/python/spacewalk/common/rhnConfig.py)

Reply:
I see. This change was merged recently, looks like `prepare-lzreposync-layout` is outdated in that regard. We can add this as a TODO for later, or you could `git rebase` your branch. Up to you.